website/docs/help-and-support/troubleshooting-guide/query-errors.md

```diff
@@ -5,6 +5,16 @@ description: >-
 # Query Errors
 The section illustrates common query errors and how to resolve them on Appsmith.
 
+## Find error messages from the datasource
+
+When a query fails, Appsmith commonly shows a notification stating something like: "UncaughtPromiseRejection: <query_name> failed to execute."
+
+![](/img/as_error.png)
```

Comment:
the screenshot seems to be little bit blurry. If you want you can use this screenshot. 
![Screenshot 2022-11-30 at 4 23 09 PM](https://user-images.githubusercontent.com/43263619/204777642-06200309-9585-4f2e-b30f-bdecbf5d3345.png)